.circleci/website.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+set -e
+
+git diff-tree --no-commit-id --name-only -r HEAD > files_changed.txt
+if ! grep -E "(^docs\/.*)|(^website\/.*)" files_changed.txt; then
+    echo "Skipping deploy & test. No relevant website files have changed"
+else
+    echo "Relevant website files have changed"
+    if [[ $CIRCLE_PROJECT_USERNAME == "facebook" && -z $CI_PULL_REQUEST && -z $CIRCLE_PR_USERNAME ]]; then
+        # configure Docusaurus bot
+        git config --global user.email "docusaurus-bot@users.noreply.github.com"
+        git config --global user.name "Website Deployment Script"
+        echo "machine github.com login docusaurus-bot password $DOCUSAURUS_PUBLISH_TOKEN" > ~/.netrc
+        # install Docusaurus and generate file of English strings
+        yarn && cd website && yarn write-translations
+        # crowdin install
+        sudo apt-get update
+        sudo apt-get install default-jre rsync
+        wget https://artifacts.crowdin.com/repo/deb/crowdin.deb -O crowdin.deb
+        sudo dpkg -i crowdin.deb
+        sleep 5
```

Comment:
is this sleep still needed?

Reply:
Tbh it's not needed but I'm just following the previous script. Want to remove ?